t/01-customers.t

```diff
@@ -18,7 +18,7 @@ subtest 'basic stuff' => sub {
 };
 
 subtest 'create customer with no data' => sub {
-    my $cust = stripe->create_customer;
+    my $cust = stripe->create_customer({});
```

Comment:
I'm not sure why this was necessary. Without it, I got:

    t/01-customers.t .. 1/? In call to WebService::Stripe::create_customer(),
    missing required argument $data at t/01-customers.t line 21.